eslint.config.mjs

```diff
@@ -41,7 +41,12 @@ export default tseslint.config(
    * Global ignores for common build and dependency directories
    */
   {
-    ignores: ["**/node_modules/**", "**/dist/**", "**/storybook-static/**"],
+    ignores: [
+      "**/node_modules/**",
+      "**/dist/**",
+      "**/storybook-static/**",
+      "**/docs-validation.generated.spec.tsx", // Auto-generated from .dev.mdx files
```

Comment:
The auto-generated file is ignored in lint (and typechecking a few files down) only because we might generate it locally, and it won't pass linting and typechecking